dolphinscheduler-remote/src/main/java/org/apache/dolphinscheduler/remote/config/NettyClientConfig.java

```diff
@@ -39,15 +40,20 @@
     private boolean soKeepalive = true;
 
     /**
-     *  send buffer size
+     * send buffer size
      */
     private int sendBufferSize = 65535;
 
     /**
-     *  receive buffer size
+     * receive buffer size
      */
     private int receiveBufferSize = 65535;
 
+    /**
+     * connect timeout millis
+     */
+    private int connectTimeoutMillis = 3;
```

Comment:
3000

Reply:
Thank you very much for your suggestion, this is indeed my mistake, 3 millis is so small